### PR TITLE
Setup of the project with Django

### LINKED ISSUE #5: django project setup
## Description
Setup the Django rest framework project for OSP and create the required directory structure.

## Acceptance Criteria
- [ ] create a base setup for Django project
- [ ] add necessary dependencies
- [ ] the server should be up and running

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 day

### PR DESCRIPTION
**Pull request by bismitaguha**
### Description

- Added `README.md` with basic setup and contributing instructions
- Added `requirements.txt` with the necessary dependencies for the project.

Fixes #5 

### Type of Change:

- Code
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
The server is up and running and the mentioned URLs in `settings.py` are working

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by bismitaguha** [comment]
@abha224 @Monal5031 @mayburgos Please review! :tada:

**Review by Monal5031 — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Django REST Framework
+
+## Setup 
+To setup the project locally go through [this wiki page](https://github.com/anitab-org/open-source-programs-web/wiki/Fork,-Clone,-Remote-and-Pull-Request).
+Make sure you have [pip](https://pip.pypa.io/en/stable/installing/) and [virtualenv](https://pypi.org/project/virtualenv/) installed.
+
+1. Database Setup
+<br>
+Before starting with the project create a db in you local using PostgreSQL with the following details. Refer to `main/settings.py` if you have any confusion.
+```
```
Comment (abha224) [inline README.md]: add a new line here, the below lines aren't formatted correctly.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Django
+- Django REST Framework
+
+## Setup 
+To setup the project locally go through [this wiki page](https://github.com/anitab-org/open-source-programs-web/wiki/Fork,-Clone,-Remote-and-Pull-Request).
+Make sure you have [pip](https://pip.pypa.io/en/stable/installing/) and [virtualenv](https://pypi.org/project/virtualenv/) installed.
+
+1. Database Setup
+<br>
+Before starting with the project create a db in you local using PostgreSQL with the following details. Refer to `main/settings.py` if you have any confusion.
```
Comment (abha224) [inline README.md]: I suggest you add a few SQL commands( eg : createdb etc)  for setting up PostgreSQL

**Comment by bismitaguha** [comment]
@abha224 I have made the respective changes in the previous commit and squashed the new one with the old commit. Please review! :100:

**Review by abha224 — APPROVED** [review]
Great work! The pr looks good to be merged.

**Pull request merged**